packages/server/src/api/bannerRouter.ts

```diff
@@ -54,6 +55,44 @@ interface PuzzlesDataResponse {
     debug?: Debug;
 }
 
+const addLocalLanguageContent = (
+    test: BannerTest,
+    variant: BannerVariant,
+    country: string,
+    moduleName: string,
+): { bannerContent?: BannerContent; mobileBannerContent?: BannerContent } => {
+    const { bannerContent, mobileBannerContent } = variant;
```

Comment:
I should of used this much simpler way of pulling out bannerContent from the variant. Will do in future. Thanks.